geps/gep-2649/index.md

```diff
@@ -0,0 +1,744 @@
+|HTTPRoute default B|Value in HTTPRoute|Value in HTTPRoute|Value in HTTPRoute|Value in HTTPRoute|
+
+
+## User discoverability and status
+
+### Standard label on CRD objects
+
+Each CRD that defines an Inherited Policy object MUST include a label that specifies
+that it is a Policy object, and that label MUST specify that the object is an
+`inherited` one.
+
+The label is `gateway.networking.k8s.io/policy: inherited`.
+
+This solution is intended to allow both users and tooling to identify which CRDs
+in the cluster should be treated as Policy objects, and so is intended to help
+with discoverability generally. It will also be used by the forthcoming `kubectl`
+plugin.
+
+### Conditions
+
+Implementations using Policy objects MUST include a `spec` and `status` stanza,
+and the `status` stanza MUST contain a `conditions` stanza, using the standard
+Condition format.
+
+Policy authors should consider namespacing the `conditions` stanza with a
+`controllerName`, as in Route status, if more than one implementation will be
+reconciling the Policy type.
+
+#### On `Policy` objects
+
+Each Inherited Policy MUST populate the `Accepted` condition and reasons
+as defined in the Go spec, a snapshot of which is included below. The canonical
+representation is in the actual Go files. (At the time of writing, this is in
+`apis/v1alpha2/policy_types.go`)
+
+This allows the responsible controller to indicate that the Policy has been
+accepted for processing.
+
+```go
+// PolicyConditionType is a type of condition for a policy.
+type PolicyConditionType string
+
+// PolicyConditionReason is a reason for a policy condition.
+type PolicyConditionReason string
+
+const (
+  // PolicyConditionAccepted indicates whether the policy has been accepted or rejected
+  // by a targeted resource, and why.
+  //
+  // Possible reasons for this condition to be True are:
+  //
+  // * "Accepted"
+  //
+  // Possible reasons for this condition to be False are:
+  //
+  // * "Conflicted"
+  // * "Invalid"
+  // * "TargetNotFound"
+  //
+  PolicyConditionAccepted PolicyConditionType = "Accepted"
+
+  // PolicyReasonAccepted is used with the "Accepted" condition when the policy has been
+  // accepted by the targeted resource.
+  PolicyReasonAccepted PolicyConditionReason = "Accepted"
+
+  // PolicyReasonConflicted is used with the "Accepted" condition when the policy has not
+  // been accepted by a targeted resource because there is another policy that targets the same
+  // resource and a merge is not possible.
+  PolicyReasonConflicted PolicyConditionReason = "Conflicted"
+
+  // PolicyReasonInvalid is used with the "Accepted" condition when the policy is syntactically
+  // or semantically invalid.
+  PolicyReasonInvalid PolicyConditionReason = "Invalid"
+
+  // PolicyReasonTargetNotFound is used with the "Accepted" condition when the policy is attached to
+  // an invalid target resource
+  PolicyReasonTargetNotFound PolicyConditionReason = "TargetNotFound"
+)
+```
```

Comment:
I've been trying to keep this PR as small as possible, since it's already enormous. But I'll add some updates to the status sections to roll your and @robscott's feedback in.

Reply:
I added some information suggesting the PolicyAffected condition, as in Direct Attached Policy, but I want us all to spend some time considering apiserver fanout  load concerns before going ahead with any more recommendations here.